atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/AbstractSchemaTest.java

```diff
@@ -0,0 +1,193 @@
+                RowResult.of(resultCell, EncodingUtils.encodeUnsignedVarLong(testValue)));
+        Mockito.when(transaction.getRows(any(), any(), any())).thenReturn(expectedResults);
+
+        long value = getSingleRowFirstColumn(transaction, testRowKey);
+
+        assertEquals(value, testValue);
+        ArgumentCaptor<Iterable> argument = ArgumentCaptor.forClass(Iterable.class);
+        Mockito.verify(transaction, times(1))
+                .getRows(eq(tableRef), argument.capture(), eq(firstColSelection));
+
+        List<byte[]> argumentRows = Lists.newArrayList(argument.getValue());
+        assertEquals(argumentRows.size(), 1);
+        assertTrue(Arrays.equals(argumentRows.get(0), PtBytes.toBytes(testRowKey)));
+    }
+
+    @Test
+    public void testGetMultipleRowsFirstColumn() {
+        AbstractTransaction transaction = mock(AbstractTransaction.class);
+        Cell expectedCell = Cell.create(PtBytes.toBytes(testRowKey), PtBytes.toBytes(firstColShortName));
+        Cell anotherExpectedCell =
+                Cell.create(PtBytes.toBytes(anotherTestRowKey), PtBytes.toBytes(firstColShortName));
+        SortedMap<byte[], RowResult<byte[]>> resultsMap = new TreeMap<>(UnsignedBytes.lexicographicalComparator());
+        resultsMap.put(
+                testRowKey.getBytes(),
+                RowResult.of(expectedCell, EncodingUtils.encodeUnsignedVarLong(testValue)));
+        resultsMap.put(
+                anotherTestRowKey.getBytes(),
+                RowResult.of(anotherExpectedCell, EncodingUtils.encodeUnsignedVarLong(anotherTestValue)));
+        Mockito.when(transaction.getRows(any(), any(), any())).thenReturn(resultsMap);
+
+        List<Long> result = getMultipleRowsFirstColumn(transaction, Arrays.asList(testRowKey, anotherTestRowKey));
+
+        assertEquals(result, Arrays.asList(testValue, anotherTestValue));
+        ArgumentCaptor<Iterable> argument = ArgumentCaptor.forClass(Iterable.class);
+        Mockito.verify(transaction, times(1))
+                .getRows(eq(tableRef), argument.capture(), eq(firstColSelection));
+
+        List<byte[]> argumentRows = Lists.newArrayList(argument.getValue());
+        assertEquals(argumentRows.size(), 2);
+        assertTrue(Arrays.equals(argumentRows.get(0), PtBytes.toBytes(testRowKey)));
+        assertTrue(Arrays.equals(argumentRows.get(1), PtBytes.toBytes(anotherTestRowKey)));
+    }
+
+    @Test
+    public void testRowRange() {
+        String endRowKey = "testRowKey3";
+        String testStringValue = "value1";
+        String anotherTestStringValue = "value2";
+
+        AbstractTransaction transaction = mock(AbstractTransaction.class);
+        Cell expectedCell = Cell.create(PtBytes.toBytes(testRowKey), PtBytes.toBytes(secondColShortName));
+        Cell anotherExpectedCell =
+                Cell.create(PtBytes.toBytes(anotherTestRowKey), PtBytes.toBytes(secondColShortName));
+        Mockito.when(transaction.getRange(any(), any())).thenReturn(
```

Comment:
nit: Replace `any()` with more specific args since we know the expected params here. Simliarly, for other tests where its easy. Not super important as we have asserts on the args later anyways.

Reply:
Thanks for the suggestion, only updated the params that we weren't capturing and checking later in the test anyway.